includes/reader-revenue/my-account/class-woocommerce-my-account.php

```diff
@@ -795,6 +795,9 @@ public static function update_payment_methods_for_all_subs() {
 	 * Whether email changes are enabled.
 	 */
 	public static function is_email_change_enabled() {
+		if ( class_exists( '\Newspack_Manager\Features' ) ) {
+			return \Newspack_Manager\Features::is_newspack_user();
```

Comment:
if the user is not a "newspack_user" we still want to enable the feature if the feature flag is on. Need to return here only if it's `true`, otherwise should check the constant

Reply:
Ah. Good catch! Fixed in https://github.com/Automattic/newspack-plugin/pull/3824/commits/c5b53aa17edf871ba7ce37f50a361461d9099468